<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserDialog</class>
 <widget class="QDialog" name="UserDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>933</width>
    <height>544</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Пользователь</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_12">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QCheckBox" name="enabledCheckBox">
           <property name="text">
            <string>Разрешен</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QDateEdit" name="birthdayDateEdit">
           <property name="displayFormat">
            <string>dd.MM.yyyy</string>
           </property>
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="maleRadioButton">
           <property name="text">
            <string>Муж.</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="femaleRadioButton">
           <property name="text">
            <string>Жен.</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="nameLabel">
           <property name="text">
            <string>Имя</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="nameLineEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="loginLabel">
           <property name="text">
            <string>Логин</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="loginLineEdit"/>
         </item>
         <item>
          <widget class="QLabel" name="passwordLabel">
           <property name="text">
            <string>Пароль</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="passwordLineEdit">
           <property name="echoMode">
            <enum>QLineEdit::PasswordEchoOnEdit</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="phone1Label">
           <property name="text">
            <string>Телефон 1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="phone1LineEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="phone2Label">
           <property name="text">
            <string>Телефон 2</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="phone2LineEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="emailLabel">
           <property name="text">
            <string>E-mail</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="emailLineEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="addressLabel">
         <property name="text">
          <string>Адрес фактический</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="addressPlainTextEdit"/>
       </item>
       <item>
        <widget class="QLabel" name="notesLabel">
         <property name="text">
          <string>Примечания</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="notesPlainTextEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <item>
          <widget class="QLabel" name="positionLabel">
           <property name="text">
            <string>Должность</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="positionLineEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="passportGroupBox">
         <property name="title">
          <string>Паспорт</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QLabel" name="pseriesLabel">
              <property name="text">
               <string>Серия</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="pseriesLineEdit"/>
            </item>
            <item>
             <widget class="QLabel" name="pnumberLabel">
              <property name="text">
               <string>Номер</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="pnumberLineEdit"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <widget class="QLabel" name="pdepartmentLabel">
              <property name="text">
               <string>Выдан</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="pdepartmentLineEdit"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="paddressLabel">
            <property name="text">
             <string>Адрес регистрации</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPlainTextEdit" name="paddressPlainTextEdit"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="rightsGroupBox">
         <property name="title">
          <string>Разрешения</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_13">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QCheckBox" name="execCartCheckBox">
              <property name="text">
               <string>Продажи</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="returnSaleCheckBox">
              <property name="text">
               <string>Возврат продажи</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="execIncomeCheckBox">
              <property name="text">
               <string>Оприходование</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="execAdminCheckBox">
              <property name="text">
               <string>Администрирование</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QCheckBox" name="execAuditCheckBox">
              <property name="text">
               <string>Учет остатков</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="execOrderCheckBox">
              <property name="text">
               <string>Заказ товара</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="webAdminCheckBox">
              <property name="text">
               <string>Доступ к веб-админ.</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="performerCheckBox">
              <property name="text">
               <string>Исполнение заказов</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QListWidget" name="officesListWidget">
       <property name="toolTip">
        <string>Офисы исполнителя заказов</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="folderPushButton">
       <property name="text">
        <string>Папка...</string>
       </property>
       <property name="icon">
        <iconset resource="kcrm.qrc">
         <normaloff>:/images/folder.png</normaloff>:/images/folder.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okPushButton">
       <property name="text">
        <string>Сохранить</string>
       </property>
       <property name="icon">
        <iconset resource="kcrm.qrc">
         <normaloff>:/images/ok32.png</normaloff>:/images/ok32.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="kcrm.qrc"/>
 </resources>
 <connections/>
</ui>
